### ISSUE TITLE
[ibm_cbr_zone] Error: Provider produced inconsistent result after apply

### ISSUE THREAD

**Issue by ocofaigh** [IBM-Cloud/terraform-provider-ibm#5590]
When running an apply in schematics on the code [here](https://github.com/terraform-ibm-modules/terraform-ibm-icd-redis/tree/main/solutions/standard), it failed with this:

```
 2024/08/29 10:19:59 Terraform apply | Error: Provider produced inconsistent result after apply
 2024/08/29 10:19:59 Terraform apply | 
 2024/08/29 10:19:59 Terraform apply | When applying changes to module.cbr_zone.ibm_cbr_zone.cbr_zone, provider
 2024/08/29 10:19:59 Terraform apply | "provider[\"registry.terraform.io/ibm-cloud/ibm\"]" produced an unexpected
 2024/08/29 10:19:59 Terraform apply | new value: Root object was present, but now absent.
 2024/08/29 10:19:59 Terraform apply | 
 2024/08/29 10:19:59 Terraform apply | This is a bug in the provider, which should be reported in the provider's own
 2024/08/29 10:19:59 Terraform apply | issue tracker.
```

### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform CLI and Terraform IBM Provider Version

- terraform 1.6.6
- `ibm-cloud/ibm v1.68.1`

### Affected Resource(s)

* ibm_cbr_zone

### Terraform Configuration Files

https://github.com/terraform-ibm-modules/terraform-ibm-icd-redis/tree/main/solutions/standard

### Debug Output

[apply-log.txt](https://github.com/user-attachments/files/16798000/apply-log.txt)

### Expected Behavior

No error

### Actual Behavior

Error: Provider produced inconsistent result after apply

### Steps to Reproduce

1. `terraform apply`

### References

* #0000

**Comment by hkantare** [comment]
@zhenwan 
Can some from team look into the panic issue

**Comment by JonahFarc** [comment]
I see you reproduced it recently, but could you reproduce it again and additionally provide the logs for this last occurrence?

**Comment by ocofaigh** [comment]
@JonahFarc Its an intermittent issue, so I have not yet reproduced with terraform provider trace logs enabled. I am running the code at https://github.com/terraform-ibm-modules/terraform-ibm-icd-redis (specifically in `module.redis.module.redis.module.cbr_rule[0].ibm_cbr_rule.cbr_rule`). We are using the module defaults.

**Comment by zhenwan** [comment]
@ocofaigh The PR to fix this issue has been merged

**Comment by ocofaigh** [comment]
Thanks, I assume its part of 1.70.0 then?

**Comment by zhenwan** [comment]
Don't know what the build plan is, I don't see this fix is built into 1.70.0 https://github.com/IBM-Cloud/terraform-provider-ibm/releases/tag/v1.70.0-beta0

**Comment by zhenwan** [comment]
@ocofaigh  yes, I see the fix is in v1.70.0:  https://github.com/IBM-Cloud/terraform-provider-ibm/compare/v1.69.2...v1.70.0. Let me know if you still see the bug. Please help close this issue if you think the bug was fixed.

<img width="1496" alt="image" src="https://github.com/user-attachments/assets/0277ea91-c487-4701-b931-5c57eabe1d2b">

**Comment by ocofaigh** [comment]
Going to close this - if we reproduce I will re-open
